assets/l10n/intl_zh_Hant.arb

```diff
@@ -527,8 +527,8 @@
         "type": "text",
         "placeholders": {}
     },
-    "downloadFile": "下載文件",
```

Comment:
because it is wrong, IMO, delete it, then people will translate it again in Weblate

Reply:
true, updated I only kept english, french and vietnamese